aggregators/codec.go

```diff
@@ -100,44 +100,17 @@ func (m *CombinedMetrics) ToProto() *aggregationpb.CombinedMetrics {
 	return pb
 }
 
-// FromProto converts protobuf representation to CombinedMetrics.
-func (m *CombinedMetrics) FromProto(pb *aggregationpb.CombinedMetrics) {
-	m.Services = make(map[ServiceAggregationKey]ServiceMetrics, len(pb.ServiceMetrics))
-	for _, ksm := range pb.ServiceMetrics {
-		var k ServiceAggregationKey
-		var v ServiceMetrics
-		k.FromProto(ksm.Key)
-		v.FromProto(ksm.Metrics)
-		m.Services[k] = v
-	}
-	if pb.OverflowServices != nil {
-		m.OverflowServices.FromProto(pb.OverflowServices)
-		m.OverflowServiceInstancesEstimator = hllSketch(pb.OverflowServiceInstancesEstimator)
-	}
-	m.EventsTotal = pb.EventsTotal
-	m.YoungestEventTimestamp = pb.YoungestEventTimestamp
-}
-
 // MarshalBinary marshals CombinedMetrics to binary using protobuf.
-func (m *CombinedMetrics) MarshalBinary() ([]byte, error) {
+// This should be the last call for the CombinedMetrics and all
+// proto resources of CombinedMetrics will be released in this call.
```

Comment:
MarshalBinary is only called in one place, by the merger. Should we just inline it there?

Reply:
Done!